javascript/app-discover/src/components/MenuBar.hooks.tsx

```diff
@@ -208,27 +181,89 @@ export function useFixedInterventionRangesToggleMenuItem() {
 	)
 }
 
-export function useTogglePauseButtonMenuItem() {
-	const [paused, { toggle: togglePaused }] = useBoolean(true)
-	const [, setPauseAutoRun] = useRecoilState(PauseAutoRunState)
+export function useAutoLayoutToggleMenuItem() {
+	const [autoLayoutEnabled, setAutoLayoutEnabled] = useRecoilState(
+		AutoLayoutEnabledState,
+	)
+	const handleClick = useCallback(async () => {
+		setAutoLayoutEnabled(true)
+		await wait(300)
```

Comment:
This is a brittle solution and may present intermittent bugs. If the layout library has a callback or promise you can subscribe to, use that. Otherwise we may want to wrap the layout in something that does have that, so we have predictable end states.